### PR TITLE
docs: sort docs files and add QA section

### PR DESCRIPTION
**Pull request by karlbaumhauer**
## Description:
Adding a new docs section to describe and link the respective VRT / QA Workflow files in Figma.

closes https://github.com/solid-design-system/figma/issues/61

## Definition of Reviewable:
- [x] Documentation is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

File: packages/components/src/docs/General/08-Testing.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Docs/General/Testing" />
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:3]: Testing is a misleading term. I would suggest `Quality Assurance`.

File: packages/components/src/docs/General/08-Testing.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## VRT / QA Workflow
+
+In order to ensure a certain level of quality and reliability, we use a combination of visual regression testing and manual QA.
+
+### Development
+
+#### Visual Regression Testing
+
+We use chromatic screenshot testing to ensure that our components are visually correct. This automatically runs on every PR and will show any visual changes that have been made. Changes can be approved or rejected and will be shown in the PR.
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:13]:
```suggestion
We use Chromatic screenshot testing to ensure that our components are visually correct. This automatically runs on every PR and will show any visual changes that have been made. Changes can be approved or rejected and will be shown in the PR.
```

File: packages/components/src/docs/General/08-Testing.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Code Analyzer
+
+We use several automations to ensure that our code is consistent and correct. This includes:
+
+- QualityGate - this action builds the whole project and veryfies the syntax of the code
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:19]:
```suggestion
- Quality Gate - this action builds the whole project, verifies linting, formatting and runs all tests
```

File: packages/components/src/docs/General/08-Testing.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- Validate PR Title - this action checks the PR title to ensure that it follows our naming conventions matching with sematic versioning
+- CodeQL Scan - this action checks the code for any security vulnerabilities
+
+The above actions are automatically run on every PR and are required to pass before the PR can be merged.
+
+#### Manual QA
+
+Besides the automatic checks, we also do manual QA on every PR. This includes:
+
+- code review
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:31]:
```suggestion
- code review with a minimum of one reviewer
```

File: packages/components/src/docs/General/08-Testing.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+We use chromatic screenshot testing to ensure that our components are visually correct. This automatically runs on every PR and will show any visual changes that have been made. Changes can be approved or rejected and will be shown in the PR.
+
+#### Code Analyzer
+
+We use several automations to ensure that our code is consistent and correct. This includes:
+
+- QualityGate - this action builds the whole project and veryfies the syntax of the code
+- File Size Check - this action checks the new bundle size of the project to ensure that we don't add too much bloat
+- Linting - this action checks the code for any linting errors
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:21]:
```suggestion
```

This check does not exist.

File: packages/components/src/docs/General/08-Testing.mdx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+#### Visual Regression Testing
+
+We use chromatic screenshot testing to ensure that our components are visually correct. This automatically runs on every PR and will show any visual changes that have been made. Changes can be approved or rejected and will be shown in the PR.
+
+#### Code Analyzer
+
+We use several automations to ensure that our code is consistent and correct. This includes:
+
+- QualityGate - this action builds the whole project and veryfies the syntax of the code
+- File Size Check - this action checks the new bundle size of the project to ensure that we don't add too much bloat
```
Comment (mariohamann) [inline packages/components/src/docs/General/08-Testing.mdx:20]:
```suggestion
- File Size Check - this action communicates and compares the bundle size to monitor performance
```

File: packages/components/src/docs/General/08-Testing.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Details
+
+A detailed description of the QA process can be found in the [VRT / QA Workflow File for Development](https://www.figma.com/file/yg2PKZ0ghndLBAPYm7kbaN/VRT-%2F-QA-Workflow?type=whiteboard&node-id=427-582&t=aJsKurM4wVpfftjE-4) in Figma.
+
+### Design
+
+#### Visual Regression Testing
+
+VRT is currently in evaluation for the design team.
```
Comment (coraliefeil) [inline packages/components/src/docs/General/08-Testing.mdx:42]: VRT using dedicated Figma plugins is currently in evaluation for the design team.

File: packages/components/src/docs/General/08-Testing.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+#### UI Review
+
+Changes in the UI are reviewed by the design team using the chromatic feature enviroment and chromatic screenshot testing mentioned above. This allows the design team to review the changes in the actual UI and not just in a design tool. The reviewer can either accept or reject the changes and leave comments.
```
Comment (coraliefeil) [inline packages/components/src/docs/General/08-Testing.mdx:46]: New/changed code components are reviewed by the design team using the Chromatic feature enviroment, Chromatic screenshot testing mentioned above and Storybook code inspection. This allows the design team to review the changes in the actual UI and not just in a design tool. The reviewer can either accept or reject the changes and leave comments. After all comments have been fixed, design reviews again and approves the PR on Chromatic if all design checks have been passed.

**Review by coraliefeil — COMMENTED** [review]
Thanks a lot, left some comments.

**Comment by karlbaumhauer** [comment]
@mariohamann @coraliefeil pls re-review

**Review by coraliefeil — APPROVED** [review]

**Pull request merged**